### PR TITLE
m1 setup

### LINKED ISSUE #87: Tensorflow - Illegal hardware instruction
When running the check for `pandas`, `scikit` and `Tensorflow` the following message comes up (WSL Student):

image

After digging into it we followed this [article](https://tech.amikelive.com/node-887/how-to-resolve-error-illegal-instruction-core-dumped-when-running-import-tensorflow-in-a-python-program/), it seems that his computer is missing AVX support. Downgrading to Tensorflow 1.5 would require Python 3.7. Building from source might not be stable, but we could try that.

For now we skipped Tensorflow installation and just in case he will run it on Google Colab. 
Any solution would be appreciated!

### PR DESCRIPTION
**Pull request by krokrob**
this add a new setup for M1, allowing running TensorFlow. Also bumps python to 3.9.6
this fixes #87

### CODE REVIEW

**Comment by krokrob** [comment]
@gmanchon it looks like there is an issue with M1 and Docker, if you could have a look at it before we merge this one please. A possible fix:  https://lewagon-alumni.slack.com/archives/C015E0C601F/p1628691338061200?thread_ts=1628073886.011800&cid=C015E0C601F

**Review by bdepebhe — COMMENTED** [review]
Since the PR is covering M1 specificities, I would recommend to include [this fix](https://blog.jaimyn.dev/how-to-build-multi-architecture-docker-images-on-an-m1-mac/) to build x86 images (compatible with cloud run) during 'predict in prediction' day.

File: build.rb [hunk 1]
```diff
@@ -116,12 +121,13 @@
   "WINDOWS.md" => DEFAULT_SUBS,
   "macOS.md" => DEFAULT_SUBS,
   "LINUX.md" => DEFAULT_SUBS,
+  "macOS_M1.md" => DEFAULT_SUBS,
```
Comment (gmanchon) [inline build.rb:124]:
```suggestion
```

**Review by gmanchon — APPROVED** [review]

**Comment by gmanchon** [comment]
@krokrob LGTM 👍

@bdepebhe we will address the possibility to build docker images for production for the M1 in the lectures and challenges. The buildx commands indeed work perfectly 🙏 :

``` bash
docker buildx build --platform linux/amd64 -t image_name --load .
docker buildx build --platform linux/amd64 -t ${GCR_MULTI_REGION}/${GCP_PROJECT_ID}/${DOCKER_IMAGE_NAME} --load .
```

**Comment by gmanchon** [comment]
@krokrob all good 🎉

File: LINUX.md [hunk 2]
```diff
@@ -496,20 +496,35 @@ pip install --upgrade pip
...
+pip install -U yapf jupyterlab seaborn plotly nbconvert xgboost statsmodels pandas-profiling dtale jupyter-resource-usage
 ```
 
-### Packages check up
+### TensorFlow
+
+Install [TensorFlow](https://www.tensorflow.org/):
+
+```bash
+pip install -U 'tensorflow<2.6'
```
Comment (brunolajoie) [inline LINUX.md:513]: Why not 2.6?

**Pull request merged**